hackathon_site/event/views.py

```diff
@@ -7,15 +7,20 @@
 from django.urls import reverse_lazy
 from django.views.generic.base import TemplateView
 from django.views.generic.edit import FormView
+from django.conf import settings
```

Comment:
Take a look at this file again, lots of unused or duplicated imports (I'm guessing from starting working in this file before switching to api_views.py)